argonaut/src/main/scala/io/finch/argonaut/Decoders.scala

```diff
@@ -10,13 +10,13 @@ import io.finch.internal.HttpContent
 trait Decoders {
 
   /**
```

Comment:
I wonder what happened to my (and others) IJ so it generates these wrong scaladoc comments. Let me see if I can find where it's configurable.